### PR TITLE
Misc updates

### PR DESCRIPTION
**Pull request by antban**
Update python version for running tests on build machine, fix test warnings

### CODE REVIEW

File: bubuku/id_extractor.py [hunk 1]
```diff
@@ -24,17 +32,11 @@ def get_broker_id(self):
...
         with open(meta_path) as f:
-            lines = f.readlines()
-            for line in lines:
-                match = re.search('broker\.id=(\d+)', line)
-                if match:
-                    self.broker_id = match.group(1)
-                    return self.broker_id
-        return None
+            self.broker_id = _search_broker_id(f.readlines())
+        return self.broker_id
```
Comment (adyach) [inline bubuku/id_extractor.py:36]: will it return None as before in case there is no match ?

Comment (adyach) [reply bubuku/id_extractor.py:36]: i see tests 👍

**Comment by adyach** [comment]
👍

File: bubuku/id_extractor.py (hunk 1 above)
Comment (antban) [reply bubuku/id_extractor.py:36]: Yes, as it did before. I was trying to understand the logic behind it. Failed and renamed the class.

**Comment by antban** [comment]
:+1:

**Pull request merged**